examples/fastapi/advanced.py

```
@@ -0,0 +1,432 @@
+# 백그라운드 작업
+# =============================================================================
+
+async def log_batch_request(
+    api_key: str,
+    message_count: int,
+    success_count: int,
+    execution_time: float
+):
+    """배치 요청 로깅 (백그라운드 작업)"""
+    # 실제로는 데이터베이스나 로깅 시스템에 저장
+    print(f"📊 배치 요청 로그:")
```

Flake8 (7.2.0)

[error] f-string is missing placeholders

(F541)